implementations/rust/ockam/ockam_core/src/error/code.rs

```diff
@@ -0,0 +1,419 @@
+//! Ockam error code enumerations.
```

Comment:
I thought we don't want to introduce coupling. Are we fine with the fact that developers of transport/vault/etc. crates won't be able to introduce their own Kind and Origin values?

Reply:
Yep, I think so.